2-mandatory/3-bush-berries.js

```diff
@@ -22,7 +22,10 @@
 */
 
 function isBushSafe(berryArray) {
-  //Write your code here
+  if (berryArray.every((currentValue) => currentValue == "pink"))
+    return "Bush is safe to eat from";
+  if (berryArray.some((currentValue) => currentValue == "pink"))
+    return "Toxic! Leave bush alone!";
 }
```

Comment:
once you've done the `every` check and it fails you dont need do the `some` check as you know this from the result of the first check. you can jus return `Toxic`